Hearthstone/app/src/main/java/com/example/hearthstone/adapter/HearthStoneAdapterSP.kt

```diff
@@ -0,0 +1,43 @@
+package com.example.hearthstone.adapter
+
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+import com.example.hearthstone.R
+import com.example.hearthstone.data.model.HSCardsByClassModel
+import com.example.hearthstone.databinding.CardByClassItemBinding
+
+class HearthStoneAdapterSP(private var cards: List<HSCardsByClassModel>) :
+    RecyclerView.Adapter<HearthStoneAdapterSP.HearthstoneViewHolder>() {
+    inner class HearthstoneViewHolder(view: View) : RecyclerView.ViewHolder(view) {
+        var binding = CardByClassItemBinding.bind(view)
+        var context = view.context
+
+        fun bindCardByClass(card: HSCardsByClassModel) {
+            binding.cardModel = card
+            if (card.img.isNullOrEmpty()) {
+                binding.imageCard.setImageResource(R.drawable.photo_copy_2)
```

Comment:
https://bumptech.github.io/glide/doc/placeholders.html
You can use glide to set a placeholder